### ISSUE TITLE
this version is not working with amd cards?

### ISSUE THREAD

**Issue by AjlaaSS** [NebuTech/NBMiner#827]
so far I have crash same moment after dag is created and mining should start
some error code than reset miner after 10 seconds

I did try with -- parameter 2 but nothing changes. it can't work

asking because some of my rigs have mixed nvidia and amd cards. is there some solution or this is expected to be patched?

tnx

**Comment by chimaabc** [comment]
Same problems. I think it is a nbminer41.0 bug. Hope the team resolves it asap.

**Comment by AjlaaSS** [comment]
> Same problems. I think it is a nbminer41.0 bug. Hope the team resolves it asap.


yes now here is stated update that they are aware of problem and we can expect it to be fixed in next updated. it really created   kind of problem for me

**Comment by rpbigpapa** [comment]
Hi i have a mixed rig 1660 supers 3060 3080 and AMD a rx5600xt and rx5700xt and after crashing a few times after dag but then takes off with 101m on 3080lhrv2 and 49m on 3060lhrv2 and both AMDshow great hash as expected

**Comment by AjlaaSS** [comment]
> Hi i have a mixed rig 1660 supers 3060 3080 and AMD a rx5600xt and rx5700xt and after crashing a few times after dag but then takes off with 101m on 3080lhrv2 and 49m on 3060lhrv2 and both AMDshow great hash as expected


I don't fully understand what you say.

You mean you get it stable using 41.3 ?
